pytest-pixl/src/pytest_pixl/data/omop-resources/batch_input.csv

```diff
@@ -1,4 +1,4 @@
-procedure_id,mrn,accession_number,project_name,extract_generated_timestamp,study_date,study_uid
-1,mrn,123,i-am-a-project,2023-01-01T00:00:00Z,2023-01-01,1.2.3
-2,mrn,234,i-am-a-project,2023-01-01T00:00:00Z,2023-01-01,2.3.4
-3,mrn,345,i-am-a-project,2023-01-01T00:00:00Z,2023-01-01,3.4.5
+procedure_id,mrn,accession_number,project_name,extract_generated_timestamp,study_date,study_uid,participant_id
```

Comment:
why don't we just call this a `pseudo_patent_id`? I also wouldn't expect for it to be set in every file but we can just have a null value.

Reply:
[8020006](https://github.com/UCLH-Foundry/PIXL/pull/484/commits/80200068fc67c56ecc0b9dca989df9c09de0abec)